cookies/views/resource.py

```diff
@@ -886,19 +886,23 @@ def dataset(request, dataset_id):
     Details about a dataset.
     """
     dataset = get_object_or_404(Dataset, pk=dataset_id)
+    resource_param = None
     if dataset.dataset_type == Dataset.EXPLICIT:
         resource_count = dataset.resources.count()
         collection_count = 0
+        resource_param = '?dataset=' + dataset_id
     else:
         collections, containers = apply_dataset_filters(request.user,
                                                         dataset.filter_parameters)
+        resource_param = '?entity_type=' + dataset.filter_parameters.split('entity_type=')[1]
```

Comment:
not sure what you are trying to do here, but this would split all filter parameters where ever 'entity_type' appears. so you'd loose everything before entity type.

Reply:
I am getting the resources of dynamic database through entity_type. The entity_type is stored in dataset table under filter_parameter attribute in the following manner - 
<b>content=&created_by=&name=&o=&entity_type=19</b>

In the above code I was  trying to extract the entity_type value. I have changed the code a bit and now, filter_parameter is converted into a dictionary first. Entity_type is then retrieved from this dict.